src/Graphics/OpenGLContext/opengl_ColorBufferReaderWithBufferStorage.cpp

```diff
@@ -30,7 +30,6 @@ void ColorBufferReaderWithBufferStorage::_initBuffers()
 	// Initialize Pixel Buffer Objects
 	for (u32 index = 0; index < m_numPBO; ++index) {
 		m_bindBuffer->bind(Parameter(GL_PIXEL_PACK_BUFFER), ObjectHandle(m_PBO[index]));
-		m_fence[index] = 0;
```

Comment:
No longer using fences since they can hurt performance and this is not really trying to be that accurate.